lib/routes/osu/beatmaps/latest-ranked.ts

```diff
@@ -0,0 +1,349 @@
+import { Data, DataItem, Route } from '@/types';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+const descriptionDoc: string = `
+Subscribe to the new beatmaps on https://osu.ppy.sh/beatmapsets.
+
+Parameters allows you to:
+
+- Filter game mode
+- Limit beatmap difficulty
+- Show/hide game mode in feed title
+
+Check out paramters description for more details.
+
+:::tip
+You could make use of \`difficultyLimit\` paramters to create a "high difficulty/low difficulty only" only feed.
+
+For example, if you only wants to play low star rating beatmap like 1 or 2 star, you could subscribe to:
+
+    /osu/latest-ranked/difficultyLimit=U2
+
+This will filter out all beatmapsets that do not provide at least one beatmap with star rating<=\`2.00\`.
+
+Similarly, you could use lower bound to filter out beatmapsets which don't have at least one beatmap
+with star rating higher than a certain threshold.
+
+    /osu/latest-ranked/difficultyLimit=L6
+
+Now all beatmapsets that don't provided at least one beatmap with star rating higher than \`6.00\` will be filtered.
+:::
+`;
+
+export const route: Route = {
+    path: '/latest-ranked/:routeParams?',
+    categories: ['game'],
+    example: '/osu/latest-ranked/includeMode=osu&includeMode=mania&difficultyLimit=L3&difficultyLimit=U7',
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+        supportRadar: true,
+    },
+    parameters: {
```

Comment:
Missing parameter description of `routeParams`
https://github.com/DIYgod/RSSHub/blob/2e0b865b1afad4d637dde07ec26785c5e92e3833/lib/routes/osu/beatmaps/latest-ranked.ts#L37